cbcontainers/state/components/sensor_daemon_set.go

```diff
@@ -445,6 +445,12 @@ func (obj *SensorDaemonSetK8sObject) mutateCndrVolumesMounts(container *coreV1.C
 		index := commonState.EnsureAndGetVolumeMountIndexForName(container, name)
 		commonState.MutateVolumeMount(container, index, fmt.Sprintf("%v", hostPath.Path), false)
 	}
+
+	// mutate mount for container-runtimes unix sockets files for the container tracking processor
```

Comment:
We should probably wrap this with
`if isCndrEnbaled(agentSpec.Components.Cndr) {`
if it's relevant only for CNDR